src/components/Banner/Banner.module.css

```diff
@@ -176,16 +199,87 @@
 
   background: var(--eds-theme-color-background-utility-error);
   color: var(--eds-theme-color-text-utility-error);
+
+  a:hover,
+  a:focus {
+    color: var(--eds-color-other-ruby-800); /* 2 */
+  }
 }
 
 /**
  * Banner brand
  * 1) The thick left border matches the icon color
+ * 2) Links inside the Banner get their color from the banner variant styles
  */
 .banner--brand {
   --border-light-color: var(--eds-theme-color-border-brand-primary-subtle);
   --border-dark-color: var(--eds-theme-color-icon-brand-primary); /* 1 */
 
   background: var(--eds-theme-color-background-brand-primary-default);
   color: var(--eds-theme-color-text-brand-primary);
+
+  a:hover,
+  a:focus {
+    color: var(--eds-color-brand-grape-800); /* 2 */
+  }
+}
+
+/*------------------------------------*\
+    # ACTION STYLES
+\*------------------------------------*/
+
+.banner__action > * {
+  /* TODO: move to mixin */
```

Comment:
This comment here is key – these styles are just copy/pasted over from the `Button` because I didn't want to put too much time into this direction in case we decide to do something else.